pontoon/terminology/utils.py

```diff
@@ -3,8 +3,76 @@
 from xml.sax.saxutils import escape, quoteattr
 
 
-def build_terminology_file(term_translations, locale):
+def build_tbx_v2_file(term_translations, locale):
     """
+    Generates contents of the TBX 2008 (v2) file (TBX-Default dialect):
```

Comment:
Do you know about command-line linters for the TBX dialects?

Reply:
There might be some here:
https://www.tbxinfo.net/tbx-downloads/
https://www.tbxinfo.net/tbx-downloads-2/